contracts/EtherSplitter.sol

```diff
@@ -5,7 +5,43 @@ pragma solidity ^0.6.0;
  * Splits transferred Ether
  */
 
-contract EtherSplitter {
-    constructor() public {
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract EtherSplitter is Ownable {
+    using SafeMath for uint256;
+
+    address payable public firstAddr;
+    address payable public secondAddr;
+
+    constructor(address payable _firstAddr, address payable _secondAddr)
+        public
+    {
+        firstAddr = _firstAddr;
+        secondAddr = _secondAddr;
+    }
+
+    function spliter() public payable onlyOwner {
+        require(msg.value >= 1, "You must send eth");
+        uint256 value = msg.value;
+
+        uint256 splitBalance = value.div(2);
+        if (value != splitBalance.mul(2)) {
+            emit remainderReturned();
+            msg.sender.transfer(value - splitBalance.mul(2));
+        }
+
+        firstAddr.transfer(splitBalance);
+        secondAddr.transfer(splitBalance);
+
+        emit foundsSplit(firstAddr, secondAddr, splitBalance);
     }
+
+    event foundsSplit(
```

Comment:
The contract is not adhering the official [Style Guide](https://docs.soliditylang.org/en/latest/style-guide.html).
e.g. events should not be declared last, events should be uppercase.